index.html

```diff
@@ -1,313 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Projeto Instagram</title>
-    <link rel="stylesheet" href="reset.css">
-    <link rel="stylesheet" href="style.css">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
-</head>
-
-<body>
-    <header>
-        <nav>
-            <ion-icon class="iconMobile" name="logo-instagram"></ion-icon>
-
-            <div class="logo">
-                <ion-icon name="logo-instagram"></ion-icon>
-                <img src="imagens/logo.png">
-            </div>
-
-            <div class="pesquisar">Pesquisar</div>
-            
-            <div class="icon">
-                <ion-icon name="paper-plane-outline"></ion-icon>
-                <div class="iconDesktop">
-                    <ion-icon name="compass-outline"></ion-icon>
-                    <ion-icon name="heart-outline"></ion-icon>
-                    <ion-icon name="person-outline"></ion-icon>
-                </div>
-            </div>
-        </nav>
-    </header>
-
-    <div class="total">
```

Comment:
Aqui seria bacana ter encapsulado na tag main, própria tag do html para quando for um conteúdo